From: Hannes Reinecke <hare@suse.de>
Date: Fri, 20 Sep 2019 13:29:04 +0200
Subject: [PATCH] blk-mq: kABI fixes for blk-mq.h
Patch-Mainline: Never, SLE15 kABI fix
References: bsc#1137959

The additional '#include' needs to be protected to avoid the
kABI checker getting confused.

Signed-off-by: Hannes Reinecke <hare@suse.com>
---
 block/blk-mq.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/block/blk-mq.h b/block/blk-mq.h
index 8db209dfab75..8bc1302adf97 100644
--- a/block/blk-mq.h
+++ b/block/blk-mq.h
@@ -2,7 +2,9 @@
 #define INT_BLK_MQ_H
 
 #include "blk-stat.h"
+#ifndef __GENKSYMS__
 #include "blk-mq-tag.h"
+#endif
 
 struct blk_mq_tag_set;
 
-- 
2.16.4

